doc/specification.md

```diff
@@ -127,9 +136,7 @@ The ABAP types are self-contained, so it is possible to work on them in any syst
 
 The JSON schema is generated based on the fields and their ABAP type specification defined in `ty_main`. Each field defined in the structure is transformed to a JSON representation using a camel case notation (e.g, field `abap_language_version` is transformed to the field `abapLanguageVersion` in the JSON schema). The ABAP type information fills the JSON schema fields `type`, `length`, `minimum`, `maximum`.
 
-One mandatory component of the type is the field `schema` which is translated to `$schema` in the JSON schema.
-Its value in the `.json` files is the link given in the field `$id` of the JSON schema.
-Furthermore, a `header` has to be supplied.
+Fields `formatVersion` and `header` are mandatory on root level.
```

Comment:
Wouldn't it be more consistent if the field in the ABAP type was called `format_version`? Then, the generator translates it to `($)formatVersion`.

Reply:
Agree.